### ISSUE TITLE
setup in office testbed

### ISSUE THREAD

**Issue by critzo** [opentechinstitute/mlab-governor-client#4]
In order to properly test and prototype changes to this system, we need to set up a proper testbed in the office, with at least two devices and a separate server instance. 
- [ ] Chris - order two additional odroids
- [ ] Fernanda/Nat - set up small vm to host a test server

**Comment by collina** [comment]
+1, against iad0t for that matter. I am a bit worried that we don't know the specs for odroid well enough to ensure that the device does that have hardware constraints. It certainly will have issues nearing 1 Gbps, but where below is a question that should be answered and answerable.

**Comment by critzo** [comment]
Odroid specs:
http://www.hardkernel.com/main/products/prdt_info.php?g_code=G141578608433&tab_idx=2

**Comment by collina** [comment]
Right, just because Odroid says its a GigE NIC doesn't mean that in the real world it can perform at 1 Gbps, could be a number of factors also involved with the performance (such as kernel driver or other things that need to be modified), hence testing.

**Comment by critzo** [comment]
You asked for the specs, I looked them up. Also, Jordan did some testing when he selected this model. In any case, yes, let's test, or feel free to recommend something else.
